charts/ocis/templates/NOTES.txt

```diff
@@ -25,28 +18,80 @@ You can get the initial "admin" administrator user password by running:
 
 kubectl -n <namespace> get secrets/admin-user --template='{{"{{"}}.data.password | base64decode{{"}}"}}'
 
-{{ if or $storageSystem $storageUsers $store $idm $search $nats }}
+{{ $noExternalUserManagement := not .Values.features.externalUserManagement.enabled -}}
+{{- $noopCache := eq .Values.cache.type "noop" -}}
+{{- $basicAuth := .Values.features.basicAuthentication -}}
+{{- $demoUsers := .Values.features.demoUsers -}}
+{{- $oidcIdpInsecure := .Values.insecure.oidcIdpInsecure -}}
+{{- $ocisHttpApiInsecure := .Values.insecure.ocisHttpApiInsecure -}}
+{{- $externalLDAPinsecure := and .Values.features.externalUserManagement.enabled .Values.features.externalUserManagement.ldap.insecure -}}
+{{- $noSMTPencryption := and .Values.features.emailNotifications.enabled (eq .Values.cache.type "noop") -}}
+
+{{ if or $noExternalUserManagement $noopCache $basicAuth $demoUsers $oidcIdpInsecure $ocisHttpApiInsecure $externalLDAPinsecure $noSMTPencryption }}
+#################################################################################
+######   WARNING: Your deployment of oCIS does not follow all best          #####
+######   practices for production deployments of oCIS.                      #####
+######                                                                      #####
+######   Following best practices are not applied:                          #####
+{{- if $noExternalUserManagement}}
+######     - `features.externalUserManagement.enabled` should be            #####
+######       set to `true`.                                                 #####
+{{- end }}
+{{- if $noopCache}}
+######     - `cache.type` should not be set to `noop`                       #####
+{{- end }}
+{{- if $basicAuth}}
+######     - `features.basicAuthentication` should be set to `false`        #####
+{{- end }}
+{{- if $demoUsers}}
+######     - `features.demoUsers` should be set to `false`                  #####
+{{- end }}
+{{- if $oidcIdpInsecure}}
+######     - `insecure.oidcIdpInsecure` should be set to `false`            #####
+{{- end }}
+{{- if $ocisHttpApiInsecure}}
+######     - `insecure.ocisHttpApiInsecure` should be set to `false`        #####
+{{- end }}
+{{- if $externalLDAPinsecure}}
+######     - `features.externalUserManagement.ldap.insecure` should         #####
+######        be set to `false`                                             #####
+{{- end }}
+{{- if $noSMTPencryption}}
```

Comment:
I don't get this part. The name implies something with encryption. But I'm missing the encryption part in the check in line 28.